data/ideas.json

```
@@ -214,6 +214,12 @@
         "description": "App for share knowledge by donationg books.",
         "level": "intermediate",
         "tags": "book-donation"
+    },
+    {
+        "title": "Major Days in History",
+        "description": "A web application containing major days in history including births, deaths, inventions, wars, etc. User can search for days, events or persons using an API.",
+        "level": "intermediate",
+        "tags": "api-requests, web-app, search"
```

The first tag is used as a unique key to match the idea and project
```suggestion
        "tags": "major-rays-in-history, api-requests, web-app, search"
```

---

I have made the changes you requested. Thank you.